0003/README.md

```diff
@@ -100,6 +100,8 @@ For example, for endpoint event groups can be connectivity, lifecycle, token (fo
 - `{event-type}` - specific event type.
 For example, EP connectivity events the types will be connected, disconnected, dormant, awake, etc.
 
+Extra subject tokens MAY be appended to the defined above prefix in specific protocol implementations.
```

Comment:
Can we say that `{event-type}` can take up multiple segments? That way we define both the meaning and the way to handle it.

Reply:
Technically, the time series name appended in 14/TSRP, has nothing to do with the event type. It serves for a granular subscription to different time series.